osi_common.proto

```diff
@@ -167,7 +173,10 @@ message MountingPosition
 // Units are [m] for radial distance and [rad] for azimuth and elevation angles. 
 // If azimuth and elevation are zero, the referenced point lies directly ahead
 // in the central viewing direction of the sensor, and the respective vector points directly in the latter.
+// A positive angle corresponds to a clockwise rotation around the respective axis, i.e. 
+// z-axis (azimuth = yaw), y-axis (elevation = pitch).
```

Comment:
This definition is ambiguous ("clockwise") if not even plain wrong ("y-axis" - I think the rotated y'-axis was intended). I propose the following definition:

``With the x-axis of the sensor frame pointing in the central viewing direction of the sensor and the senor frame being right-handed, azimuth and elevation are defined as the rotations that would have to be applied to the sensor frame to make its x-axis point towards the referenced point or to align it with the referenced vector. The rotations are to be performed \b azimuth \b first (around the z-axis) and \b elevation \b second (around the new y-axis) to follow the definition of Orientation3D. For the sense of each rotation, the right-hand rule applies.``

Moreover, I would recommend stating the definition first and the example (lines 174 and 175) second.